### PR TITLE
devloped cart controller class

### PR DESCRIPTION
**Pull request by rashmi966**
#31

### CODE REVIEW

File: backend/src/main/java/com/moshop/backend/controller/CartController.java
Comment (smruti-banty) [inline backend/src/main/java/com/moshop/backend/controller/CartController.java:1]: /cart should be /api/v1/cart

**Review by smruti-banty — COMMENTED** [review]
@RequestMapping("/api/v1/cart") Should be like this

Change according to comment

File: backend/src/main/java/com/moshop/backend/controller/CartController.java
Comment (Sikha02) [reply backend/src/main/java/com/moshop/backend/controller/CartController.java:1]: All Changes done

File: backend/src/main/java/com/moshop/backend/services/CartService.java
Comment (Abinashkjcjanscinanconanc) [inline backend/src/main/java/com/moshop/backend/services/CartService.java:1]: remove Space from line no 16

Comment (Abinashkjcjanscinanconanc) [reply backend/src/main/java/com/moshop/backend/services/CartService.java:1]: apply resolve Conversation

**Review by Abinashkjcjanscinanconanc — COMMENTED** [review]
Change according to comments

**Review by smruti-banty — APPROVED** [review]
LGTM

**Pull request merged**